lib/erl_docgen/priv/dtd/xhtml1-frameset.dtd

```diff
@@ -208,7 +208,7 @@
 
 <!ENTITY % fontstyle.extra "big | small | font | basefont">
 
-<!ENTITY % fontstyle.basic "tt | i | b | u
```

Comment:
Don't update the xhtml1 files as they are copies of standard files and should remain as the original.

Reply:
8444a3b613

Reply:
https://www.w3.org/TR/xhtml1/DTD/xhtml1-frameset.dtd, mentioned by the file, internally, has 

```
   $Revision: 1.1 $
   $Date: 2018/03/20 03:13:13 $
```